components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss

```diff
@@ -44,10 +48,6 @@ chef-icon.SKIPPED {
   &.UKNOWN, &.SKIPPED {
     border-left-color: $status-unknown;
   }
-  &.hydrated { 
-    visibility: visible;
-    opacity: 1;
-  }
```

Comment:
no need to manually hydrate.  I haven't seen an instance so far where this was needed manually but I don't know much about it.

Reply:
ya this was me trying hard to force it to show up